CMakeLists.txt

```
@@ -205,6 +207,20 @@ if(CMAKE_CXX_VISIBILITY_PRESET)
   list(APPEND DEFAULT_CMAKE_ARGS ${CMAKE_CXX_VISIBILITY_PRESET})
 endif()
 
+if(THEROCK_BUNDLE_SYSDEPS)
+  if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
+    find_program(PATCHELF patchelf)
```

```suggestion
    find_program(PATCHELF patchelf REQUIRED)
```
Could use that here but than your more verbose error message would not be printed.

---

thanks - I'll keep the more verbose error.